src/containers/ToDoList/ToDoList.jsx

```diff
@@ -103,54 +111,77 @@ export const ToDoList = () => {
     ? tasks.filter(taskFilters[filteringType])
     : tasks;
 
+  const handleChangeMode = () => {
+    setIsIconChanging((prevIcon) => !prevIcon);
+    setIsDarkTheme((prevTheme) => !prevTheme);
```

Comment:
You could use name `prevTheme` if what you were keeping in that var would actually be a name of the theme. You're keeping just a true/false, so probably sth similar to `prevState` would be a better option.